app/src/main/java/com/example/calculator/util/Constants.kt

```diff
@@ -10,7 +10,16 @@ object Constants {
     const val FIRST_OPERAND = "firstOperand"
     const val SECOND_OPERAND = "secondOperand"
     const val OPERATOR = "operator"
+    const val ENUM = "enum"
     const val COMA = ","
     const val DOT = "."
     const val PATTERN = "#.##"
+    const val DIVISION_BY_ZERO_KEY = "division_by_zero"
+    const val INCOMPLETE_OPERATION_KEY = "incomplete_operation"
+    const val INVALID_NUMBER_KEY = "invalid_number_"
```

Comment:
The last underscore is needed or it can be removed?

Reply:
Removed last underscore.